### PR TITLE
Tracking Slugs

### PR DESCRIPTION
**Pull request by Geevies**
* Added tracking slugs, shotgun rounds that contain tracking implants, to be shot into things.
* Added a box of four tracking slugs to the armory. Allowed RnD to produce them as well.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

File: code/game/objects/items/weapons/implants/implant.dm [hunk 1]
```diff
@@ -78,11 +78,17 @@
 	else
 		..()
 
+
```
Comment (NonQueueingMatt) [inline code/game/objects/items/weapons/implants/implant.dm:81]:
```suggestion
```

File: code/game/objects/items/weapons/implants/implant.dm [hunk 2]
```diff
@@ -78,11 +78,17 @@
...
+var/global/tracking_id = 1
 /obj/item/implant/tracking
 	name = "tracking implant"
 	desc = "Track with this."
 	var/id = 1.0
 
+/obj/item/implant/tracking/Initialize()
+	. = ..()
+	id = tracking_id
+	tracking_id++
```
Comment (NonQueueingMatt) [inline code/game/objects/items/weapons/implants/implant.dm:91]: You can use a factor of rounded world.time here to do away with this global

**Review by JohnWildkins — APPROVED** [review]
do what matt said and what i suggested in discord

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**